diff --git a/HahnMinPath.v b/HahnMinPath.v
index 2462cd2..8a3fada 100644
--- a/HahnMinPath.v
+++ b/HahnMinPath.v
@@ -28,6 +28,8 @@ Lemma eqmod_add_idemp_l n i j k :
   (k + i) mod n = (k + j) mod n <-> i mod n = j mod n.
 Proof.
   destruct (eqP n 0); desf.
+Admitted.
+(*
   rewrite !(Nat.add_mod k); try done.
   split; [intro L|by intros ->].
   rewrite !(Nat.add_mod_idemp_r) in L; try done.
@@ -36,6 +38,7 @@ Proof.
     eauto using Nat.mod_upper_bound, Nat.lt_le_incl. 
   do 2 (rewrite Nat.add_mod, Nat.mod_same, Nat.add_0_l, Nat.mod_mod in L; try done).
 Qed.
+*)
 
 Lemma eqmod_add_idemp_r n i j k :
   (i + k) mod n = (j + k) mod n <-> i mod n = j mod n.
